src/app/pages/journal/journal.page.ts

```diff
@@ -188,11 +188,11 @@ export class JournalPage extends BasePageComponent implements OnInit {
   ionViewDidEnter(): void {
     this.store$.dispatch(journalActions.JournalViewDidEnter());
 
-    if (super.isIos()) {
-      // this.screenOrientation.unlock();
-      // this.insomnia.allowSleepAgain();
-      // this.deviceProvider.disableSingleAppMode();
-    }
+    // if (super.isIos()) {
```

Comment:
can we un-comment the if again, screenOrientation and insomnia functions will probably both be needed in first Journal release